zrml/styx/src/benchmarks.rs

```diff
@@ -0,0 +1,37 @@
+#![allow(
+  // Auto-generated code is a no man's land
+  clippy::integer_arithmetic
+)]
+#![allow(clippy::type_complexity)]
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+use crate::Config;
+#[cfg(test)]
+use crate::Pallet as Styx;
+use frame_benchmarking::{benchmarks, impl_benchmark_test_suite, whitelisted_caller};
+use frame_support::{
+    dispatch::UnfilteredDispatchable,
+    traits::{Currency, EnsureOrigin},
+};
+use frame_system::RawOrigin;
+use sp_runtime::SaturatedConversion;
+
+benchmarks! {
```

Comment:
You should use four spaces for indentation. In case you're wondering: RustFmt cannot reformat code written inside a macro for correctness reasons. :wink:

Reply:
You should also indent everything inside `benchmarks!` with four instead of two spaces.